### ISSUE TITLE
Browse Server Files

### ISSUE THREAD

**Issue by ramachandranv** [samvera/browse-everything#223]
Hi I have a requirement like for uploading a file instead of choosing a file from my local filesystem, I would like to browse the files in rails server. Basically I have a attachments where I should be able to view the files and insert the item like how ckeditor provides option to browse and include files in editor

**Comment by escowles** [comment]
@ramachandranv Yes, you can configure a server directory as a source of files in Browse Everything, here's an example of the configuration for that: https://github.com/pulibrary/figgy/blob/master/config/browse_everything_providers.yml#L6-L7
